safety_limiter/test/src/test_safety_limiter.cpp

```diff
@@ -268,7 +268,56 @@ TEST_F(SafetyLimiterTest, SafetyLimitLinear)
         en = true;
       publishSinglePointPointcloud2(0.5, 0, 0, "base_link", ros::Time::now());
       publishWatchdogReset();
-      publishTwist(vel, (i % 3) * 0.01);
+      publishTwist(vel, ((i % 3) - 1.0) * 0.01);
```

Comment:
looks better to make `i` range double to keep originally tested range
```suggestion
      publishTwist(vel, ((i % 5) - 2.0) * 0.01);
```